internal/config/development.yaml

```diff
@@ -1,5 +1,5 @@
 server:
-  port: 8080
+  port: 8181
```

Comment:
As we use `traefik` to balance the services, even with the same port of other services it'd work. But I agree with this change because sometimes we don't want to execute it using the stack.

Reply:
Gotcha it! Therefore, I change the commit message for this commit